src/main.c

```diff
@@ -24,7 +24,7 @@ void title() {
     printf("\n");
     printf("╔═══════════════════════════╗\n");
     printf("║ Input Labs Oy.            ║\n");
-    printf("║ Alpakka - firmware 0.88.3-dev ║\n");
+    printf("║ Alpakka - firmware 0.89.0-dev ║\n");
```

Comment:
Version should be extracted to compile time defined macro dependent on last git tag in the tree.
If last tagged commit is not current commit or code base has pending changes, add "-dev".

See [git describe](https://git-scm.com/docs/git-describe) for more info.